<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>RSX</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>187</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>NTScout</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>426</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionUpdate"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionReport"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUpdate">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/update</normaloff>:/icons/update</iconset>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionReport">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/bug</normaloff>:/icons/bug</iconset>
   </property>
   <property name="text">
    <string>Report a bug</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/info</normaloff>:/icons/info</iconset>
   </property>
   <property name="text">
    <string>About NTScout</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/settings</normaloff>:/icons/settings</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>nextClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>backClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUpdate</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>settingsTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReport</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>reportTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>nextClicked()</slot>
  <slot>backClicked()</slot>
  <slot>updateTriggered()</slot>
  <slot>settingsTriggered()</slot>
  <slot>reportTriggered()</slot>
  <slot>aboutTriggered()</slot>
 </slots>
</ui>
